frontend/src/components/Collections/components/Grid/components/Row/DatasetRow/components/UploadStatus/index.tsx

```diff
@@ -1,24 +1,19 @@
-import { Button, Intent } from "@blueprintjs/core";
```

Comment:
We remove the "Cancel" button next to the loader, since we have a new inline delete button

<img width="453" alt="Screen Shot 2021-03-31 at 12 17 08 PM" src="https://user-images.githubusercontent.com/6309723/113198699-150edd00-921b-11eb-93f9-e041bc30ed34.png">